### PR TITLE
Updates Sam Miller component (simple version)

### PR DESCRIPTION
**Pull request by samcoder140**

### CODE REVIEW

File: src/bossy.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ import {BossyFormInputError} from './components/form-input-error';
 import {TammiFleetExampleComponent} from './components/tammi-name';
 import {MyronKantExampleComponent} from './components/myron';
 import {LukeShortExampleComponent} from './components/luke-short-name-example';
+import {SamMillerComponent} from './components/sam-miller-name-example';
```
Comment (glawson) [inline src/bossy.ts]: Should the component be named 'SamMillerExampleComponent', keeping with the pattern as the others?

File: src/bossy.ts [hunk 2]
```diff
@@ -25,6 +26,7 @@ import {TylerComponent} from './components/tylercomponent';
 		MyronKantExampleComponent,
 		LukeShortExampleComponent,
 		TammiFleetExampleComponent,
+		SamMillerComponent,
```
Comment (glawson) [inline src/bossy.ts]: SamMillerExampleComponent?

File: src/bossy.ts [hunk 3]
```diff
@@ -35,6 +37,7 @@ import {TylerComponent} from './components/tylercomponent';
...
```
Comment (glawson) [inline src/bossy.ts]: SamMillerExampleComponent?

File: src/components/sam-miller-name-example.ts [hunk 4]
```diff
@@ -4,6 +4,4 @@ import {Component} from '@angular/core';
 	selector: 'sam-miller',
 	template: `<h1>Sam Miller</h1>`
 })
-export class SamMillerComponent {
-	isLoggedIn: boolean = false;
-}
+export class SamMillerComponent { name = 'Sam Miller'; }
```
Comment (glawson) [inline src/components/sam-miller-name-example.ts]: SamMillerExampleComponent?

**Comment by samcoder140** [comment]
Changed them to SamMillerExampleComponent

**Pull request merged**